server/jest-controller/list.js

```diff
@@ -12,31 +12,42 @@ module.exports = async () => {
     jest.stderr.on("data", (data) => {});
 
     jest.on("close", (code) => {
-      const orderedTests = [
-        "toml",
-        "info",
-        "sep10",
-        "deposit",
-        "withdraw",
-        "transaction",
-        "transactions",
-        "fee.optional",
-        "interactive-flows.optional",
-      ];
+      
+      const orderedTests = {
+        SEP24: [
+          "toml",
+          "info",
+          "sep10",
+          "deposit",
+          "withdraw",
+          "transaction",
+          "transactions",
+          "fee.optional",
+          "interactive-flows.optional",
+        ],
+        DIRECT_PAYMENT: ["dummy"]
+      }
```

Comment:
You can simplify this a bit by selecting the right test cases up here instead of in multiple places below

```
let testProject = process.env.PROJECT ? process.env.PROJECT : "SEP24";
const orderedTests = {
  SEP24: [...],
  DIRECT_PAYMENT: [...]
}[testProject]
```

Now orderedTests holds just the ordered tests you want, and you don't need to select them at line 39 and 50 below